tests/clients/http/wrapper.go

```diff
@@ -284,5 +284,9 @@ func (w *Wrapper) Host() string {
 }
 
 func (w *Wrapper) GetNodeIdentity(ctx context.Context) (immutable.Option[identity.PublicRawIdentity], error) {
-	return w.node.DB.GetNodeIdentity(ctx)
+	return w.client.GetNodeIdentity(ctx)
+}
+
+func (w *Wrapper) AssignNodeIdentity(ctx context.Context, ident identity.Identity) error {
+	return w.client.AssignNodeIdentity(ctx, ident)
 }
```

Comment:
question: We want to be able to assign a node identity through Http?

Reply:
that's what I also wanted to discuss on stand up

Reply:
okie dokes can leave it for then